### PR TITLE
Add Gui automation stage

### PR DESCRIPTION
**Pull request by nrusinko**
This PR intends to do the following:
- Add `DIRECTOR-GUI` stage in gitlab-ci.yml.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -563,10 +613,10 @@ cluster-cleanup:
 ## user cluster cleanup
 user-cluster-cleanup:
   when: always
-  image: harshshekhar15/gitlab-job:v1
+  image: harshshekhar15/gitlab-job:v3
   dependencies:
-    - cluster-setup-check
+    - cluster-create
```
Comment (harshshekhar15) [inline .gitlab-ci.yml]: `user-cluster-cleanup` job will depend on `cluster-setup-check`

Comment (nrusinko) [reply .gitlab-ci.yml]: This is fixed

Comment (nrusinko) [reply .gitlab-ci.yml]: @harshshekhar15 we have added only 4 stages that are currently passing and we want to merge them in master as a first step. We are going to fix and push the other 4 stages after they are fixed.

Comment (harshshekhar15) [reply .gitlab-ci.yml]: @nrusinko ok great. Thanks.

**Pull request merged**